elements/document/collection/nuxeo-collection-view-layout.html

```diff
@@ -46,9 +46,7 @@
                    empty-label="[[i18n('collections.noDocument')]]"
                    empty-label-when-filtered="[[i18n('collections.noDocumentWhenFiltered')]]"
                    hide-content-view-actions>
-      <div slot="selectionActions">
-        <nuxeo-slot slot="COLLECTION_ACTIONS" model="[[actionContext]]"></nuxeo-slot>
-      </div>
+      <nuxeo-slot slot="COLLECTION_ACTIONS" model="[[actionContext]]"></nuxeo-slot>
```

Comment:
After ELEMENTS-1012, the `slot` attribute changed to `name`, so we can use the native `slot` attribute for native slots. Without it, this `nuxeo-slot` will not be inserted in the `selectionActions` native slot:
```suggestion
      <nuxeo-slot name="COLLECTION_ACTIONS" slot="selectionActions" model="[[actionContext]]"></nuxeo-slot>
```